app/lib/activity_map.rb

```diff
@@ -0,0 +1,32 @@
+class ActivityMap
+  def initialize(m)
+    @m = m
+  end
+
+  def benchmarks
+    @m.keys.map { |k| BenchmarkId.from_s k }.sort
+  end
+
+  def capacities
+    benchmarks.map(&:capacity)
+  end
+
+  def capacity_benchmarks(capacity)
+    capacity_ = Integer(capacity)
```

Comment:
is there a different name we can use for the underscored variants, or would it work to inline with `capacity.to_i`? on first glance, it's not obvious to me what the difference is.

Reply:
I'll come up with a better name.

I found the difference between `Integer(capacity)` and `capacity.to_i` is that the `.to_i` method will return 0 on invalid strings. Typically what I would prefer to see is an exception thrown so that we can detect actually invalid data.

Reply:
ahh, that's good to know!